conf/pacific/cephadm/3-node-cluster-multiple-subnets.yaml

```diff
@@ -0,0 +1,27 @@
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        networks:
+          - subnet1
+          - subnet3
```

Comment:
Can we use actual network names? Would it make sense to have an indicator about the environment?

Reply:
Yes. But thought that logical names would be helpful in,

1) Ability to select different sub-net, if provided network has no IP address(es).
2) And with the dynamic selection of sub-nets, the `fail at gathering sources` would be reduced.

Reply:
As discussed, will go with actual network names.